src/helix-ui/styles/components/alerts.less

```diff
@@ -5,10 +5,6 @@ hx-alert {
   font-weight: 300;
   line-height: 1.25rem;
 
-  &:not(:last-child) {
-    margin-bottom: 1rem;
```

Comment:
Potentially breaking change. Removed bottom margin from `<hx-alert>`. I can't think of any scenario where the margin would be desirable, at present.

If margin is desired, we'll handle it with the a11y updates, given that we may need to wrap `<hx-alert>` elements in another container in order to apply proper `role` attributes.

Reply:
⚠️ Make sure to mention this in release notes.